src/index.rs

```diff
@@ -663,27 +665,32 @@ impl Index {
       .collect::<Result<Vec<InscriptionId>>>()?;
 
     if ids.len() > 1 {
-      let re_id_to_seq_num = rtx.open_table(REINSCRIPTION_ID_TO_SEQUENCE_NUMBER)?;
-      ids.sort_by_key(
-        |inscription_id| match re_id_to_seq_num.get(&inscription_id.store()) {
-          Ok(Some(num)) => num.value() + 1, // remove at next index refactor
+      let inscription_id_to_entry = rtx.open_table(INSCRIPTION_ID_TO_INSCRIPTION_ENTRY)?;
+      ids.sort_by_key(|inscription_id| {
+        match inscription_id_to_entry
+          .get(&inscription_id.store())
+          .unwrap()
```

Comment:
Remove unwrap

Reply:
You could build a vec of sequence numbers with a for loop, which would be fallible and return errors, then zip together with the list of ids and sort.

Reply:
You can also unwrap if there's no entry, since that's an invariant.